doc/release_notes/release_notes_2.2.rst

```diff
@@ -49,7 +49,8 @@ Elkhart Lake and Tiger Lake processor support.
     Time Coordinated Computing and Time Sensitive Networking
   * Support for functional safety with new features e.g., Intel Safety Island
 
-**On Elkhart Lake, ACRN can boot using `Slim Bootloader <https://slimbootloader.github.io/>`_ (an alternative bootloader to UEFI BIOS).**
+On Elkhart Lake, ACRN can boot using Slim Bootloader
+  `Slim Bootloader <https://slimbootloader.github.io/>`_ is an alternative bootloader to UEFI BIOS
```

Comment:
Not a biggie, but it's more appropriate to talk about an UEFI **firmware**, BIOS is the term for the legacy boot firmware found on systems.